e2e/browser/test-app/components/accessGrants/index.tsx

```diff
@@ -39,13 +42,13 @@ import { useRouter } from "next/router";
 const session = getDefaultSession();
 const SHARED_FILE_CONTENT = "Some content.\n";
 
-export default function AccessGrant({
+export default function AccessController({
   setErrorMessage,
 }: {
   setErrorMessage: (msg: string) => void;
 }) {
   const [accessGrant, setAccessGrant] = useState<AccessGrant>();
-  const [accessRequest, setAccessRequest] = useState<string>();
+  const [accessRequest, setAccessRequest] = useState<AccessRequest | string>();
```

Comment:
Let's not have an ambiguous type like this, and either keep it a plain string or an `AccessRequest` object.